src/main/java/io/cdap/plugin/gcp/gcs/sink/GCSBatchSink.java

```diff
@@ -493,8 +485,8 @@ public String getPath() {
     }
 
     @Override
-    public FileFormat getFormat() {
-      return FileFormat.from(format, FileFormat::canWrite);
```

Comment:
you are dropping `::canWrite` here. Is it intended?

Reply:
yes, this method is unused code.